viadot/utils.py

```diff
@@ -18,6 +18,42 @@
 from .exceptions import APIError
 from .signals import SKIP
 
+import json
+
+from prefect_azure import AzureKeyVaultSecretReference
+
+
+def get_credentials(credentials_block_name: str) -> dict:
```

Comment:
This should not be here as viadot is decoupled from `prefect`.

Reply:
Removed